### PR TITLE
Updates for fs2 v3

### PR DESCRIPTION
**Pull request by rehanone**
- Updated `scala-2.13 ` from 2.13.1 to 2.13.5

*Dependencies Updates*
- Updated `fs2` dependency from 2.5.0 to 3.0.2
- Updated `scala-collection-compat` dependency from 2.3.2 to 2.4.3
- Updated `commons-net` dependency from 3.7.2 to 3.8.0

*Plugin Updates*
- Updated `sbt-scoverage` plugin from 1.6.1 to 1.7.0
- Updated `sbt-sonatype` plugin from 3.9.5 to 3.9.7

### CODE REVIEW

File: src/main/scala/fs2/ftp/UnsecureFtp.scala [hunk 1]
```diff
@@ -72,19 +70,17 @@ final private class UnsecureFtp[F[_]: ConcurrentEffect: ContextShift](
         )
 
   def execute[T](f: JFTPClient => T): F[T] =
-    blocker.delay[F, T](f(unsafeClient))
+    Sync[F].delay(f(unsafeClient))
```
Comment (regis-leray) [inline src/main/scala/fs2/ftp/UnsecureFtp.scala:73]: Sync[F].delay is not equivalent to blocker.delay

please use `Sync[F].blocking()`

**Review by regis-leray — CHANGES_REQUESTED** [review]
great thank you :)

**Comment by rehanone** [comment]
@regis-leray , Fixed the Sync[F].blocking. Their is one readFile test failing which I am not sure why. Any insight on that would be much helpful. Thanks!

**Comment by regis-leray** [comment]
@rehanone thank you maybe a bug in fs2 ? try to open a bug 
i found a reference to this problem
https://github.com/typelevel/fs2/pull/1161#discussion_r195890105

```
[info]   java.lang.ClassCastException: class [Ljava.lang.Object; cannot be cast to class [B ([Ljava.lang.Object; and [B are in module java.base of loader 'bootstrap')
[info]   at fs2.Chunk.toByteBuffer(Chunk.scala:327)
```

**Comment by regis-leray** [comment]
@rehanone please try new release https://github.com/typelevel/fs2/releases/tag/v3.0.3
it should work now.
thank you

**Comment by rehanone** [comment]
@regis-leray , I have updated the PR, please review. Thanks.

**Review by regis-leray — APPROVED** [review]

**Pull request merged**